packages/functional-tests/package.json

```diff
@@ -12,14 +12,14 @@
     "test-production": "playwright test --project=production"
   },
   "devDependencies": {
-    "@playwright/test": "^1.28.1",
+    "@playwright/test": "^1.29.0",
```

Comment:
Just curious was a playwright update required for this change?

Reply:
Nah, just wanted to add it since `@playwright` dep was at 1.29